maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -406,6 +416,51 @@ class Orchestra(
         false
     }
 
+    private fun assertVisualCommand(command: AssertVisualCommand): Boolean {
+        val baseline = command.baseline + ".png"
+        val thresholdDifferencePercentage = (100 - command.thresholdPercentage).toDouble()
+
+        val screenshotsDir = File(".maestro/visual_regression").apply { mkdirs() }
+
+        val actual = File(screenshotsDir, baseline)
+
+        val expected = File
+            .createTempFile("screenshot-${System.currentTimeMillis()}", ".png")
+            .also { it.deleteOnExit() }
+
+        maestro.takeScreenshot(expected.sink(), false)
+
+        if (!actual.exists()) {
+            expected.copyTo(actual, overwrite = true)
+            return true
+        }
+
+        val photoNow: BufferedImage = ImageIO.read(expected)
+        val oldPhoto: BufferedImage = ImageIO.read(actual)
```

Comment:
I feel we should ignore the system status bar as scope of this PR since it would start to fail with difference in time or having notifications 🤔 ? WDYT?